pocs/scheduler/observation.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, field, exptime=120 * u.second, min_nexp=60,
                 (default: {10})
             priority {int} -- Overall priority for field, with 1.0 being highest
                 (default: {100})
+            filter_name {str} -- Name of the filter to be used. If specified,
+            will override the default filter name (default: {None}).
```

Comment:
```suggestion
            will override the default filter name.
```

Reply:
@lspitler I kept to the style of the previous argument descriptions. What is the reason for changing this?

Reply:
Oh I see, nm, rejecting this suggestion.